File: app/src/main/java/com/codemybrainsout/imageviewer/model/Exif.java
package com.codemybrainsout.imageviewer.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by ahulr on 08-06-2017.
 */

public class Exif implements Serializable {

    @SerializedName("make")
    @Expose
    private String make;
    @SerializedName("model")
    @Expose
    private String model;
    @SerializedName("exposure_time")
    @Expose
    private String exposureTime;
    @SerializedName("aperture")
    @Expose
    private String aperture;
    @SerializedName("focal_length")
    @Expose
    private String focalLength;
    @SerializedName("iso")
    @Expose
    private Integer iso;

    public Integer downloads;
    public Integer likes;

    public String getMake() {
        return make;
    }

    public void setMake(String make) {
        this.make = make;
    }

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    public String getExposureTime() {
        return exposureTime;
    }

    public void setExposureTime(String exposureTime) {
        this.exposureTime = exposureTime;
    }

    public String getAperture() {
        return aperture;
    }

    public void setAperture(String aperture) {
        this.aperture = aperture;
    }

    public String getFocalLength() {
        return focalLength;
    }

    public void setFocalLength(String focalLength) {
        this.focalLength = focalLength;
    }

    public Integer getIso() {
        return iso;
    }

    public void setIso(Integer iso) {
        this.iso = iso;
    }

    public Integer getDownloads() {
        return downloads;
    }

    public void setDownloads(Integer downloads) {
        this.downloads = downloads;
    }

    public Integer getLikes() {
        return likes;
    }

    public void setLikes(Integer likes) {
        this.likes = likes;
    }
}

File: app/src/main/java/com/codemybrainsout/imageviewer/model/Exif.kt
package com.codemybrainsout.imageviewer.model

import android.os.Parcelable
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

/**
 * Created by ahulr on 08-06-2017.
 */
@Parcelize
data class Exif(
    @SerializedName("make")
    @Expose
    var make: String? = null,

    @SerializedName("model")
    @Expose
    var model: String? = null,

    @SerializedName("exposure_time")
    @Expose
    var exposureTime: String? = null,

    @SerializedName("aperture")
    @Expose
    var aperture: String? = null,

    @SerializedName("focal_length")
    @Expose
    var focalLength: String? = null,

    @SerializedName("iso")
    @Expose
    var iso: Int? = null,
    var downloads: Int? = null,
    var likes: Int? = null
) : Parcelable